message-index/messages/GHC-94817/tab/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: A file that uses a tab for indentation
+---
+## Message
+```
+Tab.hs:5:1: warning: [-Wtabs] [GHC-94817]
+    Tab character found here.
+    Suggested fix: Please use spaces instead.
+  |
+5 |         x + y
+  | ^^^^^^^^
+```
+
+## Explanation
+
+A tab character was found, which can be confusing.
```

Comment:
```suggestion
A tab character was found; this can be confusing because GHC interprets the length of a tab character differently to a text editor might.
```

I think something saying why that it's an issue is more useful here?

Reply:
Don't you think it makes sense in the context of the overall description, where this reasoning is already mentioned?

If we still had examples on separate pages, then I don't think this would be redundant, but with the structure of the page being what it is, I think it is a bit redundant.

Thoughts?

Reply:
Hm, yeah I do agree. Though I'm not sure who the confusion is aimed at; is it confusing to GHC? to the programmer? iif it is indeed confusing to the programmer (because it's not interpreted as intended), then it feels like it should say why it's confusing

Reply:
How do you feel about the update that I just pushed?